workspace-loader/src/json-rpc/websocket-client.ts

```diff
@@ -93,12 +93,20 @@ export class WebsocketClient implements ICommunicationClient {
         }
     }
 
+    private sleep(ms) {
```

Comment:
private async sleep(ms: number): Promise<void>
?

Reply:
I think there is no need to put `async` here. I added some type information.